js/src/ui/Form/AddressSelect/addressSelect.js

```diff
@@ -613,6 +617,10 @@ class AddressSelect extends Component {
       focusedItem: null,
       inputValue: value
     });
+
+    if (apiutil.isAddressValid(value)) {
```

Comment:
Quite true, thought so. It's imported twice as `apiutil` and once as `util` (for use with `isAddressValid`) though